src/update-players.R

```diff
@@ -25,7 +27,28 @@ build_players <- function() {
       dplyr::distinct(gsisId, .keep_all = TRUE) |>
       tibble::as_tibble(.name_repair = janitor::make_clean_names) |>
       dplyr::arrange(display_name) |>
-      dplyr::mutate(headshot = gsub("\\{formatInstructions\\}", "f_auto,q_auto", headshot))
+      dplyr::mutate(
+        headshot = gsub("\\{formatInstructions\\}", "f_auto,q_auto", headshot),
+        weight = dplyr::case_when(
+          weight == '' ~ NA_integer_,
+          weight == 0 ~ NA_integer_,
+          T ~ as.integer(weight)
+        )
+      ) |>
+      tidyr::separate(
+        height,
+        into = c('feet', 'inches'),
+        sep = '-',
+        convert = TRUE
+      ) |>
+      dplyr::mutate(
+        height = dplyr::case_when(!is.na(feet) & is.na(inches) ~ feet,
+                                  T ~ feet * 12 + inches),
+        height = dplyr::case_when(height == 0 ~ NA_integer_,
+                                  T ~ height)
+      ) |>
```

Comment:
eyeballing this logic - if feet is 5 and inches is NA, this set of casewhens would return...5 ?

Reply:
```r
nflreadr::load_players() |>
    dplyr::pull(height) |>
    table()
#> 
#>       0-0 5-10 5-11  5-5  5-6  5-7  5-8  5-9  6-0  6-1 6-10  6-2  6-3  6-4  6-5 
#>    4    1 1246 1597    1   14   78  224  593 2086 2257    2 2601 2741 2282 1704 
#>  6-6  6-7  6-8  6-9   66   67   68   69   70   71   72   73   74   75   76   77 
#>  928  390  117   36    1    1    3    4   18   23   37   33   43   46   29   38 
#>   78   79   80 
#>   17    4    1
```

Based on what the API returns, I don't think you'd have a situation where `feet = 5` and `inches = NA`. 

This is definitely a little hardcoded for my liking, but I think it does fit what numbers we know are valid

Reply:
shouldn’t it be feet * 12 instead of just feet I think is my question

Reply:
Oh wait I see, it’s got 66 etc in the feet column

Reply:
https://tidyr.tidyverse.org/reference/separate.html Use the fill arg = left to have it in the inches column